dashboard/components/reports.py

```
@@ -0,0 +1,23 @@
+import json
+from typing import Dict, List
+from datetime import datetime
+
+class ReportGenerator:
+    """Generate reports from dashboard data"""
+    
+    @staticmethod
+    def generate_json_report(data: Dict) -> str:
+        """Generate JSON report"""
+        report = {
+            'timestamp': datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)